package.json

```diff
@@ -9,18 +9,28 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "moralis": "^1.8.0",
-    "next": "12.1.5",
-    "react": "18.0.0",
-    "react-dom": "18.0.0",
-    "react-moralis": "^1.4.0",
-    "web3uikit": "^0.1.159"
+    "moralis": "^1.3.2",
```

Comment:
why are we downgrading the moralis version?

Reply:
This version worked with absolutely no errors for me lemme try using a higher version and see if there are conflicts with the other packages